### ISSUE TITLE
"Cannot modify properties of a WrappedNative" when trying to open a URL to a text file.

### ISSUE THREAD

**Issue by stroobandt** [brython-dev/brython#422]
This issue is related to the now closed [issue 308](https://github.com/brython-dev/brython/issues/308).
The code there proposed only works for reading a JSON entity at a certain URL.
Attempts to do similarly with text files at a certain URL systematically fail.
```python
url = 'http://lgdc.uml.edu/common/DIDBGetValues?ursiCode=DB049&charName=foF2'
print(url)
info = open(url).read()
print(info)
```
Entering above code on http://brython.info/tests/editor.html?lang=en results in the following cryptic error, when working with the Pale Moon browser v26.2.1 (x64) on Xubuntu 14.04 LTS:
```
[Exception... "Cannot modify properties of a WrappedNative"
nsresult: "0x80570034 (NS_ERROR_XPC_CANT_MODIFY_PROP_ON_WN)"
location: "JS frame :: http://brython.info/src/py_exceptions.js :: $B.exception :: line 253"  data: no]
```
Am I missing something here, or is this truly a bug?

**Comment by stroobandt** [comment]
Using the Vivaldi v1.1.453.47 (Stable channel) (64-bit) browser on Linux, results in a somewhat different error message:
```
Internal Javascript error: NetworkError: Failed to execute 'send' on 'XMLHttpRequest': Failed to load 'http://lgdc.uml.edu/common/DIDBGetValues?ursiCode=DB049&charName=foF2?foo=156'.
  module 'editor' line 97
        exec(src, ns)
<completed in 813.00 ms>
```

**Comment by stroobandt** [comment]
I am closing this issue, since it is not a Brython issue.
For those interested, further information and workarounds are available at issue #308 .
